### DISCUSSION TITLE
SoundSwitch consistently loses audio outputs after PC has been asleep - Windows 11

### DISCUSSION THREAD

**Discussion by Blake-V-Walker** [Belphemur/SoundSwitch#1127] [Q&A]
Whenever PC has been asleep for a period of time (could be minimum 15-20 minutes), SoundSwitch loses the audio outputs in the hotbar and the only fix is to right-click the icon in taskbar > Playback Devices. Only once Playback Devices has been opened do the audio outputs appear and can be used again. 

Windows 11 Pro - Version 10.0.22621 Build 22621
SoundSwitch - Version (6.5.4.0)

**Comment by ryanneve** [comment]
I'm having this same problem. To temporarily fix it, I open the NVIDIA control panel, Click on Open Sound settings, and my devices are back in about 30 seconds.
Edit:
SoundSwitch 6.7.2.0
Windows 11 22H2
